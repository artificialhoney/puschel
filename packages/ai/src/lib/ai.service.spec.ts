describe('ai.service', () => {
  it('should be true', () => {
    expect(true).toMatchSnapshot();
  });
});
