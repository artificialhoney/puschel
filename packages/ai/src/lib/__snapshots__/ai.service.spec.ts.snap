// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ai.service should be true 1`] = `true`;
